### ISSUE TITLE
Tests failing: incorrect call to NLopt

### ISSUE THREAD

**Issue by matbesancon** [JuliaOpt/Cbc.jl#51]
While testing the package using  `Pkg.test`, got the following error after the testset  "Quadratics":
```julia
QP solve through NL pathway with NLopt.NLoptSolver(:LD_SLSQP, NaN, 1.0e-7, NaN, 1.0e-7, nothing, 1.0e-7, 0, 0, nothing, 0, nothing, 0): Error During Test
  Got an exception of type ArgumentError outside of a @test
  ArgumentError: invalid NLopt arguments
  Stacktrace:
   [1] chk(::Int32) at /home/mbesancon/.julia/v0.6/NLopt/src/NLopt.jl:259
   [2] chks(::Int32) at /home/mbesancon/.julia/v0.6/NLopt/src/NLopt.jl:276
   [3] optimize!(::NLopt.Opt, ::Array{Float64,1}) at /home/mbesancon/.julia/v0.6/NLopt/src/NLopt.jl:529
   [4] optimize!(::NLopt.NLoptMathProgModel) at /home/mbesancon/.julia/v0.6/NLopt/src/NLoptSolverInterface.jl:203
   [5] #solvenlp#165(::Bool, ::Function, ::JuMP.Model, ::JuMP.ProblemTraits) at /home/mbesancon/.julia/v0.6/JuMP/src/nlp.jl:1271
   [6] (::JuMP.#kw##solvenlp)(::Array{Any,1}, ::JuMP.#solvenlp, ::JuMP.Model, ::JuMP.ProblemTraits) at ./<missing>:0
   [7] #solve#116(::Bool, ::Bool, ::Bool, ::Array{Any,1}, ::Function, ::JuMP.Model) at /home/mbesancon/.julia/v0.6/JuMP/src/solvers.jl:172
   [8] solve(::JuMP.Model) at /home/mbesancon/.julia/v0.6/JuMP/src/solvers.jl:150
   [9] macro expansion at /home/mbesancon/.julia/v0.6/JuMP/test/nonlinear.jl:150 [inlined]
   [10] macro expansion at ./test.jl:921 [inlined]
   [11] macro expansion at /home/mbesancon/.julia/v0.6/JuMP/test/nonlinear.jl:141 [inlined]
```

It seems that NLopt expects different arguments

**Comment by mlubin** [comment]
This is an unfortunate consequence of https://github.com/JuliaOpt/JuMP.jl/issues/849 and Cbc using JuMP's tests to test itself. It will be resolved in the transition to JuMP/MathOptInterface.

**Comment by mlubin** [comment]
Resolved by #61
